setup.py

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env python2
 
-from distutils.core import setup
+from setuptools import setup
+
 import bayesian_changepoint_detection
 
-setup(name='bayesian_changepoint_detection',
-      version=bayesian_changepoint_detection.__version__,
-      description='Some Bayesian changepoint detection algorithms',
-      author='Johannes Kulick',
-      author_email='johannes.kulick@ipvs.uni-stuttgart.de',
-      url='http://github.com/hildensia/bayesian_changepoint_detection',
-      packages = ['bayesian_changepoint_detection'],
-      requires=['scipy', 'numpy']
-     )
+setup(
+    name='bayesian_changepoint_detection',
+    version=bayesian_changepoint_detection.__version__,
+    description='Some Bayesian changepoint detection algorithms',
+    author='Johannes Kulick',
+    author_email='johannes.kulick@ipvs.uni-stuttgart.de',
```

Comment:
Since you are updating it here anyway, can you change this to mail@johanneskulick.net, thanks.

Reply:
I am happy to do this, but you've already merged. Or do you mean when I update to `setup.cfg`?